src/components/EditCodeDialog.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  Flex,
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalOverlay,
+} from "@chakra-ui/react";
+import { EditorProject } from "@microbit-foundation/react-editor-embed";
+import { ReactNode } from "react";
+import Editor from "./Editor";
+
+interface EditCodeDialogProps {
+  isOpen: boolean;
+  toolbarItemsLeft?: ReactNode;
+  toolbarItemsRight?: ReactNode;
```

Comment:
Might as well simplify and drop these and the extraToolbars prop below.

Reply:
[f77ad94](https://github.com/microbit-foundation/ml-trainer/pull/290/commits/f77ad94d5000abfecdd0de3ce4edaa0b3a74a043)